views/index.erb

```diff
@@ -0,0 +1,14 @@
+<!-- 'Hello World' -->
+<html style="background-color: #faf49e", align="center">
+<form>
+  <label for ="peep"> Enter peep here:</label><br>
+<!-- <input type="text"  id="peep" name"Enter Peep" ><br> -->
+<textarea id="peep" name="Enter Peep" rows="5"></textarea><br>
+<input type="submit" value="Enter Peep">
+</form>
+<ul >
+  <% @peep.each do |peeps| %>
+    <li><%= peeps %></li>
+  <% end %>
+</ul>
+</html>
```

Comment:
Well written and clear code to display peeps. This is easy for anyone to change if it needs to be going forward. Great work.